packages/rs-dpp/src/state_transition/state_transitions/identity/identity_create_transition/state_transition_like.rs

```diff
@@ -46,4 +48,10 @@ impl StateTransitionLike for IdentityCreateTransition {
             IdentityCreateTransition::V0(transition) => transition.owner_id(),
         }
     }
+
+    fn asset_lock(&self) -> Option<&AssetLockProof> {
```

Comment:
I wouldn't pollute a general interface with non-general methods. We have a separate trait `AssetLockProved`

Reply:
We agreed that I will make a PR on top of this PR to propose another solution